docs/source/deployment/azure.md

```diff
@@ -0,0 +1,5 @@
+# Azure ML pipelines
+
+## `kedro-azureml` plugin
+
+For deployment to Azure ML pipelines, you should [consult the documentation](https://kedro-azureml.readthedocs.io/en/0.3.6/) for the [`kedro-azureml` plugin](https://github.com/getindata/kedro-azureml) from Get In Data | Part of Xebia that enables you to run your code on Azure ML Pipelines in a fully managed fashion.
```

Comment:
* `GetInData | Part of Xebia`
* Maybe embed video tutorial? https://www.youtube.com/watch?v=w_9RzYpGplY